library.properties

```diff
@@ -8,4 +8,4 @@ url=https://github.com/sa-fx/WindsOfChange
 category=Sensors
 architectures=*
 includes=WindsOfChange.h
-depends=SD, OneWire
\ No newline at end of file
+depends=SD, OneWire (=2.3.0), ModbusMaster
```

Comment:
The latest version of OneWire, 2.3.7, has a typo which prevents compilation, so the library is intentionally rolling back to 2.3.0